### PR TITLE
Updated launchers

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
The M3 smoke grenade uses smokebomb drop now; M202 FLASH is off-limit for NPCs and SIMULTANEOUS; survivor's rocket launcher gained BACKBLAST; MODE_BURST and RELOAD_ONE was dropped from many launchers.

### CODE REVIEW

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -667,7 +667,8 @@
         "range" : 140,
         "dispersion" : 30,
         "recoil" : 45,
-        "effects" : ["COOKOFF", "SMOKE_BIG", "NEVER_MISFIRES"]
+        "drop": "smkebomb_act",
```
Comment (mugling) [inline data/json/items/ammo.json]: Not sure this is a good idea

**Comment by mugling** [comment]
Launchers could do with some attention so this is a welcome PR but you might want to consider how to make the items useful because currently they have no actual tactical value

**Comment by BorkBorkGoesTheCode** [comment]
I think launcher damage could be modelled after the tankbot main gun.

**Comment by mugling** [comment]
> I think launcher damage could be modelled after the tankbot main gun

Can you expand on this. I can add some code support if you can clearly outline a design plan?

**Comment by BorkBorkGoesTheCode** [comment]
Okay. I should be able to post a plan within a week.

**Comment by BorkBorkGoesTheCode** [comment]
@mugling I have just tested two launcher ammos with the EXPLOSIVE flag against a square of zombies: the target zombie took damage, but the surrounding zombies were unaffected. The same test with EXPLOSIVE_BIG caused the death of many zombies.

**Comment by BorkBorkGoesTheCode** [comment]
Two thoughts: Explosions could apply similar effects to the BEANBAG flag, and inflict bleeding conditions to the torso of mundane living things.

**Comment by BorkBorkGoesTheCode** [comment]
Temporarily closed.

**Pull request closed without merging**